recipes-bsp/startup-scripts/files/cypress_bluetooth.sh

```diff
@@ -19,7 +14,6 @@ sensortag_smack_workaround()
 
 do_start()
 {
-    attach_cypress_bt
     sensortag_smack_workaround
```

Comment:
Previously, the btattach happened before we did the smack workaround. Since we are now being driven by an asynchronous kernel event coming up netlink - shouldn't the smack workaround be in the ttyUSB up script as once we have created the device node and then attached the BT adapter attach?

Reply:
The only thing that really matters is that the smack rules are applied before the bluetoothSensorTag app starts trying to talk to DBus.  So there's no problem.